client/my-sites/marketing/tools/index.tsx

```diff
@@ -92,6 +97,22 @@ export const MarketingTools: FunctionComponent = () => {
 			<MarketingToolsHeader handleButtonClick={ handleBusinessToolsClick } />
 
 			<div className="tools__feature-list">
+				<MarketingToolsFeature
+					title={ translate( 'Get more traffic by promoting your content' ) }
+					description={ translate(
+						'Promote your content on millions of sites across WordPress.com and Tumblr with our new promoted post tool. Get new users to your best content, spending less than you would on a cup of coffee..'
```

Comment:
I think the double period at the end of the sentence should be changed to a single period?

```suggestion
						'Promote your content on millions of sites across WordPress.com and Tumblr with our new promoted post tool. Get new users to your best content, spending less than you would on a cup of coffee.'
```

Reply:
Updated.